widgetastic_manageiq/__init__.py

```diff
@@ -3329,3 +3329,18 @@ def get_first_entity(self):
             return self.entity_class(parent=self, name=name)
 
         raise ItemNotFound("No Entities found on this page")
+
+
+class RadioWidget(Widget):
```

Comment:
I've slightly updated RadioGroup - https://github.com/ManageIQ/integration_tests/pull/5567/files#diff-264354a78406ae4ca6d836bd7388d376R1803

Please check whether this fix suits you. I suspect this your new widget isn't necessary

Reply:
It looks like your modifications should work. I'll try them out once your PR is merged. Thanks!